dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/report/TrackerBundleImportReportTest.java

```diff
@@ -202,8 +202,22 @@ public void testSerializingAndDeserializingImportReport()
         assertEquals( toSerializeReport.getBundleReport().getStats().getTotal(),
             deserializedReport.getBundleReport().getStats().getTotal() );
 
-        assertEquals( toSerializeReport.getBundleReport().getTypeReportMap().get( TrackerType.TRACKED_ENTITY ),
-            deserializedReport.getBundleReport().getTypeReportMap().get( TrackerType.TRACKED_ENTITY ) );
+        assertEquals(
+            toSerializeReport.getBundleReport().getTypeReportMap().get( TrackerType.TRACKED_ENTITY ).getTrackerType(),
```

Comment:
Extract `toSerializeReport.getBundleReport().getTypeReportMap().get( TrackerType.TRACKED_ENTITY )` and `deserializedReport.getBundleReport().getTypeReportMap().get( TrackerType.TRACKED_ENTITY )` to variables will improve readability

Reply:
I thought the same but forgot to do it, thank you